### PR TITLE
Referencing again the library wrapper

### PR DESCRIPTION
**Pull request by marqdevx**

### CODE REVIEW

File: content/tutorials/portenta-h7/por-ard-gs/content.md [hunk 1]
```diff
@@ -60,7 +60,8 @@ In this step you will check if Windows is able to detect the Portenta H7. To do
 ### 4. Uploading the Classic Blink Sketch
 Let's program the Portenta with the classic blink example to check if the connection to the board works:
```
Comment (sebromero) [inline content/tutorials/portenta-h7/por-ard-gs/content.md:61]:
```suggestion
Let's program the Portenta with the classic blink example to check if the connection to the board works. There are two ways to do that:
```

File: content/tutorials/portenta-h7/por-ard-gs/content.md [hunk 2]
```diff
@@ -60,7 +60,8 @@ In this step you will check if Windows is able to detect the Portenta H7. To do
...
 
--   In the classic Arduino IDE open the blink example by clicking the menu entry File->Examples->01.Basics->Blink.
+-   In the classic Arduino IDE open the blink example by clicking the menu entry File->Examples->01.Basics->Blink (Small will be required).
```
Comment (sebromero) [inline content/tutorials/portenta-h7/por-ard-gs/content.md:63]:
```suggestion
-   In the classic Arduino IDE open the blink example by clicking the menu entry File->Examples->01.Basics->Blink. You need to swap LOW and HIGH pin values as the built-in LED on Portenta is turned on by pulling it LOW.
```

File: content/tutorials/portenta-h7/por-ard-gs/content.md [hunk 3]
```diff
@@ -60,7 +60,8 @@ In this step you will check if Windows is able to detect the Portenta H7. To do
...
+-   By downloading the 'Arduino_Pro_Tutorials' library and going to File->Examples->Arduino_Pro_Tutorials->Setting Up Portenta H7 For Arduino->Blink
```
Comment (sebromero) [inline content/tutorials/portenta-h7/por-ard-gs/content.md:64]:
```suggestion
-   By downloading the 'Arduino_Pro_Tutorials' library and opening the pre-made sketch under File->Examples->Arduino_Pro_Tutorials->Setting Up Portenta H7 For Arduino->Blink 
```

**Review by sebromero — CHANGES_REQUESTED** [review]
I'd add a bit more details to that as suggested.

**Comment by sebromero** [comment]
@marqdevx Can you sign the CLA please? Check the link above

**Review by marqdevx — COMMENTED** [review]
LGTM

**Pull request merged**